src/main/steps/respondent/task-list/content.ts

```diff
@@ -260,7 +260,6 @@ export const generateContent: TranslationFn = content => {
     content.userCase?.caseTypeOfApplication === 'C100'
       ? getC100Banners(content.userCase, translations, content.userIdamId)
       : getFl401Banners(content.userCase, translations, content.userIdamId);
-  console.log(JSON.stringify(buildProgressBarStages(content.userCase!)));
```

Comment:
Please remove this

Reply:
I removed it that's why it's showing that